README.md

```diff
@@ -1,2 +1,89 @@
 # variant-remapping
-The pipeline for remapping VCF variants between two arbitrary FASTA assemblies.
+## Description
+The pipeline for remapping VCF variants between two arbitrary FASTA assemblies. No chain file required.
+
+**Method**: creates reads from the flanking sequences of each variant, then maps them to the new assembly using bowtie2.
+
+Currently, it only supports SNPs, and not indels.
+
+Prerequisites:
+- reverse_strand.py, included (reverse strand allele correction, uses bamnostic)
```

Comment:
Does this mean the alleles are converted to the forward strand? Could it be written down that way, for the not-so-biologically savvy (most people in the team 😂)?

Reply:
Yes! I've added an explanation in the README file.